domains/jesusvaladez.json

```
@@ -0,0 +1,11 @@
+{
+    "description": "Use to show a personal portfolio",
+    "repo": "https://github.com/JesusVala/JesusValadez.github.io.git" ,
+    "owner": {
+        "username": "JesusVala",
+        "email": "jesusvaladezf@outlook.com"
+    },
+    "record": {
+        "CNAME": "JesusValadez.github.io"
```

site isnt developer related, you could make this on a nested subdomain
```suggestion
        "CNAME": "jesusvaladez.github.io"
```